src/dc_receive_imf.rs

```diff
@@ -2262,7 +2257,7 @@ async fn dc_add_or_lookup_contacts_by_address_list(
     origin: Origin,
     prevent_rename: bool,
 ) -> Result<Vec<ContactId>> {
-    let mut contact_ids = BTreeSet::new();
+    let mut contact_ids = HashSet::new();
```

Comment:
why is that change regarded as a "downside"? 

imu, the HashSet is probably small anyway and all other things around populating BTreeSet or HashSet will probably take most of the time.

Reply:
I said this elsewhere, but in theory a small BTree with numbers is faster to build than an equivalent bunch of hash buckets.  Mostly because the overhead of computing the hash for the numbers is supposedly relatively large (but constant time) and this is not needed for the BTree.  And assuming the number of allocations will be similar, which they probably will be.